### PR TITLE
docs: React Quickstart

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: docs/docs/quickstarts/react.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+This Integration guide shows you the recommended way to integrate **ZITADEL** into your React Application.
+It demonstrates how to add a user login to your application and fetch some data from the user info endpoint.
+
+At the end of the guide you should have an application able to login a user and read the user profile.
+
+## Setup Application and get Keys
+
+Before we can start building our application we have do do a few configuration steps in ZITADEL Console.
+You will need to provide some information about your app. We recommend creating a new app to start from scratch. Navigate to your [Project](https://console.zitadel.ch/projects) and add a new application at the top of the page.
+Select Web Application and continue.
```
Comment (livio-a) [inline docs/docs/quickstarts/react.md:14]: I know react apps can be Web Apps, but aren't most just regular SPA?

Comment (livio-a) [reply docs/docs/quickstarts/react.md:14]: image would have to change as well

File: docs/docs/quickstarts/react.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+We recommend that you use [Authorization Code](../apis/openidoauth/grant-types#authorization-code) in combination with [Proof Key for Code Exchange](../apis/openidoauth/grant-types#proof-key-for-code-exchange) for all web applications.
+
+![Create app in console](/img/angular/app-create-light.png)
+
+### Redirect URLs
+
+A redirect URL is a URL in your application where ZITADEL redirects the user after they have authenticated. Set your url to the domain the web app will be deployed to or use `http://localhost:3000/` because this will be the default of npm.
```
Comment (livio-a) [inline docs/docs/quickstarts/react.md:21]: web app...

File: docs/docs/quickstarts/react.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+This library helps integrating ZITADEL Authentication in your React Application.
+
+### Create and configure Auth Module
+
+With the installed oidc pakage you will need an AuthProvider which should contain the OIDC configuration.
+
+The oidc configuration should contain **openid**, **profile** and **email** as scope and **code** as responseType.
+In the code below the authority is already set to the issuer of zitadel.ch you can find this in the ZITADEL Console on you application.
+Replace the clientId value 'YOUr-CLIENT-ID' with the generated client id of you application in ZITADEL Console.
```
Comment (livio-a) [inline docs/docs/quickstarts/react.md:60]:
```suggestion
Replace the clientId value 'YOUR-CLIENT-ID' with the generated client id of you application in ZITADEL Console.
```

File: docs/docs/quickstarts/react.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+### Run application
+
+Start your react application with the following command
+
+```bash
+npm start
+```
+
+Your browser should automatically open the app site or just go to `http://localhost:3000/`
+On opening the app in the browser you will be redirected to the login of zitadel.ch
```
Comment (livio-a) [inline docs/docs/quickstarts/react.md:105]:
```suggestion
Your browser should automatically open the app site or just go to `http://localhost:3000/`.
On opening the app in the browser you will be redirected to the login of zitadel.ch
```

File: docs/docs/quickstarts/react.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+After successfully authenticating your user you will get back to you application.
+It should show a popup which says You logged in {FirstName} {LastName}
+
+## Completion
+
+You have successfully integrated ZITADEL in your React Application!
+
+### Whats next?
+
+Now you can proceed implementing our APIs to include Authorization. Refer to our [Docs](../apis/apis) or checkout our Console Code on [Github](https://github.com/caos/zitadel) which is using GRPC to access data.
```
Comment (livio-a) [inline docs/docs/quickstarts/react.md:115]: does linking to the console code (angular) make sense?

File: docs/docs/quickstarts/react.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+Start your react application with the following command
+
+```bash
+npm start
+```
+
+Your browser should automatically open the app site or just go to `http://localhost:3000/`
+On opening the app in the browser you will be redirected to the login of zitadel.ch
+After successfully authenticating your user you will get back to you application.
```
Comment (livio-a) [inline docs/docs/quickstarts/react.md:106]:
```suggestion
After successfully authenticating your user, you will get back to you application.
```

File: docs/docs/quickstarts/react.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+It should show a popup which says You logged in {FirstName} {LastName}
```
Comment (livio-a) [inline docs/docs/quickstarts/react.md:107]: i'd highlight the `You logged in` more either using `:` bold text or as code block

File: docs/docs/quickstarts/react.md (hunk 5 above)
Comment (fgerschwiler) [reply docs/docs/quickstarts/react.md:115]: no

**Review by livio-a — APPROVED** [review]

**Pull request merged**